packages/explorer/src/app/api/table/route.ts

```diff
@@ -0,0 +1,32 @@
+import { Hex } from "viem";
+import { getDatabase } from "../database";
+
+export const dynamic = "force-dynamic";
+
+export type TableConfig = {
+  address: Hex;
+  id: string;
+  key_schema: Record<string, string>;
+  last_error: string | null;
+  name: string;
+  namespace: string;
+  schema_version: number;
+  table_id: Hex;
+  value_schema: Record<string, string>;
+};
+
+export async function GET(req: Request) {
+  const { searchParams } = new URL(req.url);
+  const tableId = searchParams.get("tableId") as Hex;
+
+  if (!tableId) {
+    return Response.json({ error: "tableId is required" }, { status: 400 });
+  }
+
+  const db = getDatabase();
+  const table = db
+    ?.prepare(`SELECT * FROM __mudStoreTables WHERE id='${tableId}'`)
```

Comment:
I think we're negating the use of prepared statements and also prone to SQL injection (because `tableId` is user-provided)
```suggestion
    ?.prepare("SELECT * FROM __mudStoreTables WHERE id=?").bind(tableId)
```

Reply:
I am unable to avoid using prepare statement with `better-sqlite3` it seems but should have parameterized SQL, now done. I was unable to parameterize `FROM` so instead checking if table name is valid i.e.

```typescript
  try {
    if (!table || !doesTableExist(table)) {
      return Response.json({ error: "table does not exist" }, { status: 400 });
    }

    const db = getDatabase();
    const query = `SELECT * FROM "${table}" LIMIT 30`;
    const rows = db?.prepare(query).all() as RowsResponse;
```